wpimath/src/main/native/include/frc/controller/PIDController.h

```diff
@@ -252,6 +252,8 @@ class WPILIB_DLLEXPORT PIDController
 
   /**
    * Returns true if the error is within the tolerance of the setpoint.
+   * The error tolerance defauls to 5%, and the error derivative tolerance
```

Comment:
The error tolerance is absolute, not a percentage. It's 0.05 of whatever unit the users is using (e.g., 0.05 inches).